01_Prereading.md

```diff
@@ -41,11 +41,11 @@ The ground state is the lowest energy state of the quantum system and is the sam
 
 Our "search space" is some subspace of all possible states of our quantum system, and the "objective function" is the energy associated with a given state. VQE makes use of a quantum system to actually measure the energy of a prepared state. Cool! So now we have a feasible way of actually computing an eigenvalue (**this is meant to be an Aha! moment by the way**).
 
-#### We're actually getting an upper bound for the ground state energy
+#### We're getting an upper bound for the ground state energy
 
-Because we're actually doing quantum measurements to get the ground state energy, we can only expect to get measurement outcomes at least as high as the ground state energy because... well that's what "ground state" is by defintion. And actually, if we're working with a practical problem which is analytically intractable, we'll never truly know it even if we do get to the exact ground state. So that's why it's called an "upper bound".
+Because we're doing the actual quantum measurements to get the ground state energy, we can only expect to get measurement outcomes at least as high as the ground state energy because... well that's what "ground state" is by defintion. And if we're working with a practical problem which is analytically intractable, we'll never truly know it even if we do get to the exact ground state. So that's why it's called an "upper bound".
```

Comment:
Happy to drop "actually" although in that case maybe your "the actual" can be dropped to. Leaving: "Because we're doing quantum measurements to...